docs/source/delta-type-widening.md

```diff
@@ -4,15 +4,34 @@ description: Learn about type widening in Delta.
 
 # Delta type widening
 
-.. note:: This feature is available in preview in <Delta> 3.2.
+.. note:: This feature is available in preview in <Delta> 3.2 and above.
 
 The type widening feature allows changing the type of columns in a Delta table to a wider type. This enables manual type changes using the `ALTER TABLE ALTER COLUMN` command and automatic type migration with schema evolution in `INSERT` and `MERGE INTO` commands.
 
 ## Supported type changes
 
-The feature preview in <Delta> 3.2 supports a limited set of type changes:
-- `BYTE` to `SHORT` and `INT`.
-- `SHORT` to `INT`
+The feature introduces a limited set of supported type changes in <Delta> 3.2 and expandes it in <Delta> 4.0 and above.
+
+.. csv-table::
+  :header: "Source type", "Supported wider types - Delta 3.2", "Supported wider types - Delta 4.0"
+
+  "`byte`","`short`, `int`","`short`,`int`,`long`, `decimal`, `double`"
+  "`short`","`int`", "`int`,`long`, `decimal`, `double`"
+  "`int`"," ","`long`, `decimal`, `double`"
+  "`long`", ,"`decimal`"
+  "`float`", ,"`double`"
+  "`decimal`", ,"`decimal` with greater precision and scale"
+  "`date`", ,"`timestampNTZ`"
+
+To avoid accidental promotion of integer values to decimals, you must manually commit type changes from `byte`, `short`, `int`, or `long` to `decimal` or `double`.
```

Comment:
"Manually commit" sounds like this would require the user to manually create and commit a transaction. Lets change this to something like "you must manually alter the type from ...".